discovery_engine/lib/src/discovery_engine_base.dart

```diff
@@ -743,6 +751,9 @@ extension _MapEvent on EngineEvent {
             _maybePassThrough(activeSearchTermRequestSucceeded),
         activeSearchTermRequestFailed:
             _maybePassThrough(activeSearchTermRequestFailed),
+        activeSearchClosedSucceeded:
```

Comment:
Having just updated this part of the code, maybe you would like to give me some feedback. I tried to auto-generate that part of the code [in this PR](https://github.com/xaynetwork/xayn_discovery_engine/pull/461), and would be curious to hear if you think that's worth it :)